src/common/cockpitpipe.c

```diff
@@ -1505,6 +1529,47 @@ cockpit_pipe_get_stderr (CockpitPipe *self)
   return priv->err_buffer;
 }
 
+gchar *
+cockpit_pipe_take_stderr_as_utf8 (CockpitPipe *self)
```

Comment:
There's no `take` left here anymore, as far as I can tell.  This is just some extra UTF8 stuff...

Reply:
It is supposed to reset the buffer to empty.  Does it not?  This is old code from cockpitpipechannel.c, read_stderr. I have factored it out into this common function.